lib/compile/util.js

```diff
@@ -214,20 +214,33 @@ function getPath(currentPath, prop, jsonPointers) {
 }
 
 
+var JSON_POINTER = /^#|\/(?:[^~]|~0|~1)*?$/;
```

Comment:
Not really: https://tools.ietf.org/html/rfc6901#section-3
It's when JSON pointer is in URI (in $ref)